examples/seismic/skew_self_adjoint/test_utils.py

```diff
@@ -1,14 +1,36 @@
 import numpy as np
 import pytest
-from devito import Grid, Function, configuration
+from devito import Grid, Function
 from examples.seismic.skew_self_adjoint import *
-configuration['language'] = 'openmp'
-configuration['log-level'] = 'ERROR'
 
 
 class TestUtils(object):
 
-    @pytest.mark.parametrize('shape', [(40, 50), (40, 50, 60)])
+    def makeGrid(self, shape, dtype):
+        origin = tuple([0.0 for s in shape])
+        extent = tuple([s - 1 for s in shape])
+        return Grid(extent=extent, shape=shape, origin=origin, dtype=dtype)
+
+    # @pytest.mark.skip(reason="temporarily skip")
+    @pytest.mark.parametrize('shape', [(41, 51), (41, 51, 61)])
+    @pytest.mark.parametrize('value', [1.0, 1.5, 2.0])
+    @pytest.mark.parametrize('dtype', [np.float32, ])
+    def test_critical_dt(self, shape, value, dtype):
+        """
+        Test the function returning CFL temporal sampling
+        """
+        tol = 1.e-5
+        grid = self.makeGrid(shape, dtype)
+        v = Function(name='v', grid=grid)
+        v.data[:] = value
+        coeff = 0.38 if len(shape) == 3 else 0.42
+        dtExpected = dtype(coeff / value)
```

Comment:
dt_expected

Reply:
section removed, as that function will be picked up from model.py soon